actions/setup-nextstrain-cli/action.yaml

```diff
@@ -29,38 +29,63 @@ inputs:
   runtime:
     description: >-
       Nextstrain runtime to configure and set as the default, as a string
-      understood by `nextstrain setup`.  Defaults to "docker".
+      understood by `nextstrain setup`.  Defaults to "docker" on Linux and
+      "conda" on macOS (where, on GitHub Actions, Docker isn't supported).  On
+      Windows, no runtime is set up by default: GitHub Actions runners can't run
+      Linux containers via WSL2 and our Conda runtime doesn't support Windows.
     type: string
-    default: docker
+    default: ${{ runner.os == 'Linux' && 'docker' || runner.os == 'macOS' && 'conda' || '' }}
     required: false
 
+  # XXX TODO: Remove this in coordination with existing callers.  Callers must
+  # stop specifying it before we remove it here in order to not break.  I've
+  # audited all of our internal usages¹ of this action to check that they don't
+  # rely on the setup-python call and prepped those usages for the removal of
+  # this input.  There are also many external usages², mostly existing in forks
+  # of our repos.
+  #   -trs, 13 Feb 2024
+  #
+  # ¹ <https://github.com/search?q=org%3Anextstrain+%2Fsetup-nextstrain-cli%40master%2F+%28path%3A*.yaml+OR+path%3A*.yml%29&type=code&p=1>
+  # ² <https://github.com/search?q=%28NOT+org%3Anextstrain%29+%2Fsetup-nextstrain-cli%40master%2F+%28path%3A*.yaml+OR+path%3A*.yml%29&type=code&p=1>
```

Comment:
It looks like all external usages are either forks or exact copies of one of our repos. I did a quick spot check and didn't find any that are actively using the GitHub workflows.

I don't think coordination with the external usages is feasible. I would be ok with dropping it now that internal usages are up to date. In the slim chance that this breaks external usage, they should be able to contact us and we can help them fix it.

Reply:
Nod. I'd be fine with that, but didn't want to start something that might cause broader fallout while I'm out. But by all means you can start it… :-)